k8s/core/limitrange.go

```diff
@@ -0,0 +1,91 @@
+package core
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: `v1` and `corev1` name aliases point  to the same import.

Reply:
fixed.